wordpress-mobile/WordPress-iOS
# Implement settings writing options

This PR implements the Default Category and Default Post Format for a Blog.
These can be set on blog settings.

How to test:

 * Try to change the values for this settings
 * Check when creating a new post in the app  if the settings are applied
 * Check if the values are in sync with the web site.

Needs Review: @koke
